google_api_lib/tasks.py

```diff
@@ -50,8 +51,12 @@ def create_google_sheets(self, puzzle_id, name, puzzle_url=None):
     sheet_url = response["webViewLink"]
     if puzzle_url:
         add_puzzle_link_to_sheet(puzzle_url, sheet_url)
-    puzzle = Puzzle.objects.filter(pk=puzzle_id).update(sheet=sheet_url)
+    puzzle = Puzzle.objects.get(pk=puzzle_id)
+    puzzle.sheet = sheet_url
+    puzzle.save()
```

Comment:
nit: i think the other way still works and is one fewer db query (update vs get + update, not 100% sure though)

Reply:
I needed the puzzle object to call `chat_room` to confirm existence.

Reply:
Oops yup, didn't read closely, sounds good